ms.html

```diff
@@ -88,7 +90,7 @@
     <script>
         window.onhashchange = onload = () => {
             const msid = window.location.hash.substr(1)
-                ;[catalogLink, letter, thumbroll, transcriptStatus, previewTranscription].forEach(el => el.setAttribute("deer-id", msid))
+                ;[catalogLink, letter, thumbroll, managedStatus, reviewedStatus, statusComment, publishedStatus, previewTranscription].forEach(el => el.setAttribute("deer-id", msid))
```

Comment:
We should probably ticket just adding a class to things we want to drop the hash id into. I've done that elsewhere and it simplifies this kind of bookkeeping and id pollution.